pages/help.py

```diff
@@ -1060,6 +1060,23 @@ def __init__(self):
         self.title = _("Suggestions")
         self.direct_link = "https://freenet.uservoice.com/"
 
+class SupportSection(Section):
+    def __init__(self):
+        self.slug = "support"
+        self.title = _("Get Support")
+    def get_content(self):
+        # License: GFDL (from old freenetproject.org website)
+        return text(md(_("""
+If you need help installing Freenet for the first time, or have trouble using Freenet and can't find an answer to your problem in the [FAQ][] above or in the [Knowledge Base][kb_link], please create a new discussion on our support forum.
```

Comment:
`[FAQ][]` needs to be `[FAQ][faq_link]` or something like that, otherwise the link will not work after translation in languages using a different abbreviation.